README.md

```diff
@@ -8,58 +8,66 @@ are showed accordingly with the response of the previous ones.
 Calling the component:
 
 ### questions.js
+
 ```javascript
 export default = [
   questionNumber: 1,
```

Comment:
I felt that you are repeating "question" in many attributes declarations, how about use it as a "marker" or a higher definition of your object? Something like that:
```javascript
question: {
   number: 1,
   type: 'twoOptionsButtons',
   text: 'Who is your favorite hero?'
  ...
}
```

Reply:
So, instead of an array we'll return an object? Something like this:

```javascript

export default {
 question: {
   number: 1,
   type: 'twoOptionsButtons',
   text: 'Who is your favorite hero?'
  ...
}
```

Reply:
Also, I liked the idea of removing the "question" part of the attributes :tada: